server/src/main/java/io/druid/server/coordination/ServerManager.java

```diff
@@ -438,7 +438,7 @@ public void dropSegment(final DataSegment segment) throws SegmentLoadingExceptio
                           }
                         },
                         new ReferenceCountingSegmentQueryRunner<T>(factory, adapter),
-                        "scan/time"
+                        "segment/scan/time"
```

Comment:
maybe we should have all query related metrics start with `query`? e.g. `query/segment/time`, `query/cache/time` to be more consistent between all the query metrics